service/controller/aws/v8/version_bundle.go

```diff
@@ -9,7 +9,7 @@ func VersionBundle() versionbundle.Bundle {
 		Changelogs: []versionbundle.Changelog{
 			{
 				Component:   "cluster-operator",
-				Description: "Add your changes here.",
+				Description: "Allow advanced CoreDNS configuration. See https://docs.giantswarm.io/guides/advanced-coredns-configuration",
 				Kind:        versionbundle.KindAdded,
```

Comment:
Past tense here too

Reply:
This doesn't make sense. This is definitely not past tense since we are allowing users to configure coredns.

Reply:
Yeah in this case, it sounds weird. Not sure if can be rephrased in past tense. Anyway will be great has a common pattern